File: ToDo/rubyonrails/6 - Ruby on Rails Piscine/4/piscine-ruby-on-rails-master/j01/ex00/var.rb
#! /usr/bin/ruby -W0

def print_obj(name, v)
	if v != nil
		puts name + " contient : " + v.to_s + " et est de type " + v.class.to_s
	else
		puts name + " contient : nil et est de type " + v.class.to_s
	end
end

def my_var()
	a = 10
	b = "10"
	c = nil
	d = 10.0
	print_obj("a", a)
	print_obj("b", b)
	print_obj("c", c)
	print_obj("d", d)
end

my_var()

File: ToDo/pythondjango/5 - Python-Django Piscine/5/python_django_piscine-master/d01/ex00/var.py
def my_var():
	s = "est de type"
	data = [42, '42', 'quarante-deux', 42.0, [42], {42: 42}, (42,), set()]
	for d in data:
		print(d, s, type(d))

if __name__ == '__main__':
	my_var()